### DISCUSSION TITLE
sofa unity api installation issues

### DISCUSSION THREAD

**Discussion by ZhihongLiao** [InfinyTech3D/SofaUnity#120] [Help - Setting up SofaUnity]
I downloaded SOFA(v24.06.00) and Unity(2022.3.40), and I followed the instruction on the github(InfinyTech3D). I downloaded the package and I checked the sofa.ini. These are ok. But when I ran the examples, Unity displayed this:
![屏幕截图 2024-09-04 003658](https://github.com/user-attachments/assets/aa5663a7-1afc-419d-9951-d7bc5d7c66f7)
![屏幕截图 2024-09-04 003753](https://github.com/user-attachments/assets/b1a069c3-51cf-47da-a63b-29ef15b6a380)
![屏幕截图 2024-09-04 003829](https://github.com/user-attachments/assets/5d26d6fb-50c0-4df3-9ec5-341ba7e9d31f)
![屏幕截图 2024-09-04 003858](https://github.com/user-attachments/assets/5adbeef1-ad01-43fa-8d6e-5f57c0140df6)
Please help me, thanks forever.

**Comment by epernod** [comment]
Hi,

sorry the v24.06 is still being released. We haven't tested the deployment yet. We plan to finish it by the end of the week.
So thanks for the test.
Could you use the [23.12](https://github.com/InfinyTech3D/SofaUnity/releases/tag/SofaUnity_v23.12) for the moment or wait until next week if you prefer to wait the v24.06.

Thank you

**Reply by ZhihongLiao** [reply]
So, you wanna let me download the sofaunity23.12? Do I need to download the older version of  SOFA? Such as SOFA(2023).

**Reply by epernod** [reply]
Hi,
in the list of archives, you have this one with SOFA DLL: https://github.com/InfinyTech3D/SofaUnity/releases/download/SofaUnity_v23.12/SofaUnity_v23.12.00_public_withDLL.unitypackage

So with this version you do not need to download SOFA apart.
Check [1st tutorial](https://youtu.be/7ucROUFBfus) if you have some doubt 

Best regards

**Reply by epernod** [reply]
Hi,

if needed the new release v24.06 installation has been tested and fixed.
Here is the official link to this new release:
https://github.com/InfinyTech3D/SofaUnity/releases/tag/SofaUnity_v24.06.00

Best regards

**Reply by epernod** [reply]
any update @ZhihongLiao
